lib/iris/tests/unit/util/test_array_equal.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2014 - 2015, Met Office
+# (C) British Crown Copyright 2014 - 2019, Met Office
```

Comment:
Did you want to follow [your suggested approach](https://github.com/SciTools/iris/pull/3272#issuecomment-468235708) and remove the year

Reply:
The template for the new file headers is here:
https://github.com/SciTools/scitools.org.uk/blob/master/documents/scitools_licensing.md#source-preamble

Reply:
I think I'd rather do it separately.

My reason is, I just realised, we are going to have to modify the licence-header check in `test_coding_standards.py` **_first_** ... see #3285

Reply:
Yep, that makes sense :+1: